Tests/Parser/Client/fixtures/feed_reader.yml

```diff
@@ -179,3 +179,123 @@
     type: feed reader
     name: Stringer
     version: ""
+-
+  user_agent: CommaFeed/2.6.0-SNAPSHOT (https://github.com/Athou/commafeed)
+  client:
+    type: feed reader
+    name: CommaFeed
+    version: 2.6.0-SNAPSHOT
+-
+  user_agent: davefeedread v0.5.25
+  client:
+    type: feed reader
+    name: davefeedread Library
+    version: 0.5.25
+-
+  user_agent: Feedbin feed-id:1669257 - 28 subscribers
+  client:
+    type: feed reader
+    name: Feedbin
+    version: ""
+-
+  user_agent: Feedly/1.0 (+https://feedly.com/poller.html; 20 subscribers; )
+  client:
+    type: feed reader
+    name: Feedly
+    version: "1.0"
+-
+  user_agent: FreshRSS/1.19.0-dev (Linux; https://freshrss.org)
+  client:
+    type: feed reader
+    name: FreshRSS
+    version: 1.19.0-dev
+-
+  user_agent: Gofeed/1.0
+  client:
+    type: feed reader
+    name: Gofeed Library
```

Comment:
Library, so should be classified as such and be named "Gofeed" without "Library"